### ISSUE TITLE
ESI v2

### ISSUE THREAD

**Issue by mattroe** [votingworks/vxsuite-roadmap#74]

**Comment by umbernhard** [comment]
Re: 

> review and maximize partitions encompassed by dm-verity, ideally leaving only /tmp and /var as writable.

This should be as simple as changing the preseed file to delete the additional non-writable partitions (I think it's just `/home`, see https://github.com/votingworks/vxsuite/issues/1899). And then testing it to make sure that that doesn't break anything. I think we did some of this exploration a while back but I can't quite remember.

**Comment by benadida** [comment]
@umbernhard agreed, this should be relatively straight-forward, though there may be some trickiness to including `/home`, figuring out where Electron is storing / caching data (should it even do that?).
